To-do list/Controllers/TaskDetailsViewController.swift

```diff
@@ -18,18 +18,33 @@ class TaskDetailsViewController: UIViewController {
     @IBOutlet private weak var titleTextField: UITextField!
     @IBOutlet private weak var descriptionTextField: UITextField!
     @IBOutlet private weak var statusButton: UIButton!
+    @IBOutlet private weak var dateTextField: UITextField!
     
     var task = Task()
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        
         prepareUIElements()
+        
+        let datePicker = UIDatePicker()
+        datePicker.datePickerMode = .date
+        datePicker.preferredDatePickerStyle = .wheels
+        datePicker.addTarget(self,
+                             action: #selector(dateChange(datePicker:)),
+                             for: UIControl.Event.valueChanged)
+        datePicker.frame.size = CGSize(width: 0, height: 200)
```

Comment:
It's better to move magic numbers to some constants

Reply:
done